core/src/main/java/hudson/cli/CommandDuringBuild.java

```diff
@@ -45,7 +45,7 @@
     protected Run getCurrentlyBuilding() throws CmdLineException {
         Run r = optCurrentlyBuilding();
         if (r==null)
-            throw new CmdLineException("This CLI command works only when invoked from inside a build");
+            throw new IllegalStateException("This CLI command works only when invoked from inside a build");
```

Comment:
Do we want to show usage in such case? Which would be an argument in favor of `CmdLineException`.